bcrs-ui/src/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface State {
+  todos: ToDo[]
+}
+
+export interface ToDo {
+  text: string
+}
```

Comment:
Nice boilerplate! Do you think this should be in a separate folder?

Reply:
100%, this was me fiddling around and i left it in. My apologies, oops. I can add the examples to the interfaces :)